Casks/v/volanta.rb

```diff
@@ -10,8 +10,13 @@
   livecheck do
     url "https://api.volanta.app/api/v1/ClientUpdate/latest-mac.yml"
     regex(%r{volanta-app/(\d+(?:\.\d+)+)-(.+)/}i)
```

Comment:
```suggestion
    regex(%r{volanta-app/v?(\d+(?:\.\d+)+)[._-](\h+)/}i)
```

While we're modifying this `livecheck` block: the existing regex is aiming to match a directory like `1.10.2-bab224b7` but `.+` could match more than what's intended, so there's something to be said for being more explicit about the suffix (to better represent our intention).